### ISSUE TITLE
Keyword Creation Wizard and IFCW button not work properly

### ISSUE THREAD

**Issue by felix-yew** [inasafe/inasafe#4383]
### Problem
I find a trouble after I git pull this morning. Nothing happen when I try to set the keyword in a layer, the KW's and IFCW's dialog box did not appear.

```
Python Console 
Use iface to access QGIS API interface or Type help(iface) for more info
Traceback (most recent call last):
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\plugin.py", line 765, in show_keywords_wizard
    from safe.gui.tools.wizard.wizard_dialog import WizardDialog
  File "C:/PROGRA~1/QGIS2~1.14/apps/qgis-ltr/./python\qgis\utils.py", line 572, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\tools\wizard\wizard_dialog.py", line 81, in <module>
    from step_kw44_fields_mapping import StepKwFieldsMapping
  File "C:/PROGRA~1/QGIS2~1.14/apps/qgis-ltr/./python\qgis\utils.py", line 572, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\tools\wizard\step_kw44_fields_mapping.py", line 17, in <module>
    from safe.gui.widgets.field_mapping_widget import FieldMappingWidget
  File "C:/PROGRA~1/QGIS2~1.14/apps/qgis-ltr/./python\qgis\utils.py", line 572, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\widgets\field_mapping_widget.py", line 14, in <module>
    from safe.gui.widgets.field_mapping_tab import FieldMappingTab
  File "C:/PROGRA~1/QGIS2~1.14/apps/qgis-ltr/./python\qgis\utils.py", line 572, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\widgets\field_mapping_tab.py", line 20, in <module>
    from parameters.qt_widgets.parameter_container import (
ImportError: cannot import name InvalidValidationException
```

### Expected Behaviour
The button can work properly and run normally to set the keyword in a layer (KW) and set the impact function (IFCW).

### Environment
* InaSAFE : 2.14.14
* QGIS : Essen
* OS : Win-7 64bit/12GB RAM
* Data used (filenames and paths so as to reproduce, a project is better):
  * Exposure : 
  * Hazard : 
  * Aggregation :
  * Extent : 

### Proposed Solution
Make it happy

Cc:
@timlinux @Charlotte-Morgan @ismailsunni @Gustry @myarjunar @samnawi @ivanbusthomi

**Comment by ismailsunni** [comment]
Can you try this fix in the wiki:
https://github.com/inasafe/inasafe/wiki/Common-Problem-in-Developing-and-Testing-InaSAFE#can-not-load-plugin-after-switching-branch-importerror-no-module-named-parameters

It looks like the same issue.

**Comment by felix-yew** [comment]
Ok, I will try it @ismailsunni

**Comment by felix-yew** [comment]
@ismailsunni - did not work, mas

**Comment by ivanbusthomi** [comment]
Hi @ismailsunni , Strangely I don't get the error and I have no problem opening the keywords wizard.

**Comment by ismailsunni** [comment]
Epilog:

Another issue after we fixed this problem with PR #4385 is like this:
```
Python Console 
Use iface to access QGIS API interface or Type help(iface) for more info
Traceback (most recent call last):
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\tools\wizard\wizard_dialog.py", line 678, in on_pbnNext_released
    new_step.set_widgets()
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\tools\wizard\step_kw44_fields_mapping.py", line 114, in set_widgets
    self.parent.layer, on_the_fly_metadata)
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\widgets\field_mapping_widget.py", line 58, in set_layer
    self.populate_tabs()
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\widgets\field_mapping_widget.py", line 79, in populate_tabs
    tab.set_layer(self.layer, self.metadata)
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\widgets\field_mapping_tab.py", line 146, in set_layer
    self.populate_parameter()
  File "C:/Users/yanuaf/.qgis2/python/plugins\inasafe\safe\gui\widgets\field_mapping_tab.py", line 258, in populate_parameter
    vertical=False
TypeError: __init__() got an unexpected keyword argument 'vertical'
```

The reason is that the user also installed another plugin that has same module name (`parameter` in http://plugins.qgis.org/plugins/EmergencyMapper/ plugin for this case). The module is not up to date, but it's loaded first (perhaps because the alphabetical order).

The solution is disabling that plugin then all will be good again.


Closing this ticket...
